inductiva/resources/resources.py

```diff
@@ -0,0 +1,62 @@
+"""Functions to manage or retrieve user resources."""
+from absl import logging
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva import resources
+
+
+def get_storage_size():
+    """Returns the total storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        instance_price = api.get_storage_size()
+        return instance_price.body
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def list():
+    """Lists all active machine group names.
+
+    Returns:
+        List of machine group names.
+    """
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        response = api.list_active_user_instance_groups()
+        if len(response.body) == 0:
+            logging.info("No active machine groups found.")
+            return response.body
+
+        machine_group_names = []
+        for machine_group in response.body:
+            machine_group_names.append(machine_group["name"])
+            logging.info("Name: %s", machine_group["name"])
+            logging.info("Nubmer of machines: %s",
+                         machine_group["num_instances"])
+            logging.info("Created at: %s", machine_group["created_at"])
```

Comment:
Should all of these be printed in the same line, to be easy to identify the number of machine groups in the logs and the info (number of machines and creation time) associated with each one?

Reply:
Yeah, I tried that but it looked messy that way. I will show it in a demo meeting, then we can decide maybe.